collection-scripts/gather-odf-consumer

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# Expect base collection path as an exported variable
+# If it is not defined, use PWD instead
+BASE_COLLECTION_PATH=${BASE_COLLECTION_PATH:-"$(pwd)"}
+
+
+STORAGE_CLIENT_NAMESPACE=$(oc get storageclient -A --no-headers | awk '{print $1}')
+INSTALL_NAMESPACE="openshift-storage"
+
+
+# collection path for OC commands
+mkdir -p "${BASE_COLLECTION_PATH}/odf-consumer_cluster/oc_output/"
+
+# Command List
+commands_get=()
+commands_get+=("pods")
+commands_get+=("subscription")
+commands_get+=("storageclients")
+commands_get+=("storageclassclaims")
+
+# YAML List
+oc_yamls=()
+oc_yamls+=("pods")
+oc_yamls+=("subscription")
+oc_yamls+=("storageclients")
+oc_yamls+=("storageclassclaims")
+
+# OC desc List
+commands_desc=()
+commands_desc+=("pods")
+commands_get+=("subscription")
+commands_desc+=("storageclients")
+commands_desc+=("storageclassclaims")
+
+
+for INSTALL_NAMESPACE in  $STORAGE_CLIENT_NAMESPACE $INSTALL_NAMESPACE ; do
+     dbglog "collecting dump of namespace ${INSTALL_NAMESPACE}"
+     { oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect ns/"${INSTALL_NAMESPACE}" --"${SINCE_TIME}" 2>&1; } | dbglog
+     dbglog "collecting dump of clusterresourceversion"
+     for oc_yaml in "${oc_yamls[@]}"; do
+          # shellcheck disable=SC2129
+          { oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect "${oc_yaml}" -n "${INSTALL_NAMESPACE}" --"${SINCE_TIME}" 2>&1; } | dbglog
```

Comment:
- iirc, client-operator is cluster scoped operator with storageclient crd at namespace scope and storageclassclaims crd at cluster scope
- as such, you should collect claims and client CRs from where ever they present however, should only collect operator related info (pods, logs, subs etc) from wherever client-operator is running only

Reply:
Yes that's correct.

Reply:
ack, thanks for the confirmation.